src/witness/witness.ml

```diff
@@ -303,7 +303,9 @@ struct
         let i_str = string_of_int i in
         match n with
         | Statement stmt  -> Printf.sprintf "s%d(%d)[%s]" stmt.sid c_tag i_str
+        | LongjmpTo stmt  -> "" (* TODO: Correct? *)
         | Function f      -> Printf.sprintf "ret%d%s(%d)[%s]" f.svar.vid f.svar.vname c_tag i_str
+        | LongjmpFromFunction f -> "" (* TODO: Correct? *)
```

Comment:
Probably not, because empty node name doesn't mean it won't be part of the ARG. But this is to be figured out after we see whether all this CFG change is strictly necessary.

Reply:
Do nodes that are not of the CFG end up in the Arg at all? Also, what would an automaton-based witness for a function using longjmp look like?

Reply:
Since they're never part of the CFG to begin with, they don't, but it's weird to have to handle them here at all.

As to what the witness would look like, probably nobody knows nor has thought of it. I guess conceptually there could be a direct edge from the `longjmp` to the `setjmp`, although in this simple manner it would violate the well-nesting of the function entry/exit edges in the witness automaton.